pkg/alertmanager/statefulset_test.go

```diff
@@ -230,7 +232,7 @@ func TestMakeStatefulSetSpecSingleDoubleDashedArgs(t *testing.T) {
 		replicas := int32(3)
 		a.Spec.Replicas = &replicas
 
-		statefulSet, err := makeStatefulSetSpec(&a, Config{})
+		statefulSet, err := makeStatefulSetSpec(&a, defaultTestConfig)
```

Comment:
I'll need some 👀 on these changes I made on the tests I hope they are 👍 

I have made the parameters in the configuration to be mandatory at this point so I need to pass over a configuration that have those in it otherwise I'll get some panics like:

```
--- FAIL: TestMakeStatefulSetSpecAdditionalPeers (0.00s)
panic: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$' [recovered]
	panic: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'

goroutine 67 [running]:
testing.tRunner.func1(0xc0002f5800)
	/usr/local/opt/go/libexec/src/testing/testing.go:792 +0x6a7
panic(0x22128e0, 0xc0001a7da0)
	/usr/local/opt/go/libexec/src/runtime/panic.go:513 +0x1b9
github.com/coreos/prometheus-operator/vendor/k8s.io/apimachinery/pkg/api/resource.MustParse(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/israel/go/src/github.com/coreos/prometheus-operator/vendor/k8s.io/apimachinery/pkg/api/resource/quantity.go:127 +0x2ae
github.com/coreos/prometheus-operator/pkg/alertmanager.makeStatefulSetSpec(0xc000436dc0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/israel/go/src/github.com/coreos/prometheus-operator/pkg/alertmanager/statefulset.go:465 +0x2a16
github.com/coreos/prometheus-operator/pkg/alertmanager.TestMakeStatefulSetSpecAdditionalPeers(0xc0002f5800)
	/Users/israel/go/src/github.com/coreos/prometheus-operator/pkg/alertmanager/statefulset_test.go:364 +0x29e
testing.tRunner(0xc0002f5800, 0x23b9880)
	/usr/local/opt/go/libexec/src/testing/testing.go:827 +0x163
created by testing.(*T).Run
	/usr/local/opt/go/libexec/src/testing/testing.go:878 +0x65a
FAIL	github.com/coreos/prometheus-operator/pkg/alertmanager	0.251s
```